lms/envs/common.py

```diff
@@ -555,25 +555,6 @@
 OPENID_PROVIDER_TRUSTED_ROOTS = ['cs50.net', '*.cs50.net']
```

Comment:
Is this needed?  We should search for OPENID as well.

Reply:
It looks like we don't need this line. Fun fact: it was added in 2012 and has not been modified since.

Reply:
I am going to punt openid removal to another PR.